app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/exporter/WSClient.java

```diff
@@ -78,8 +79,8 @@ public void pushSurveys(SurveyContainerWSObject surveyContainerWSObject,
             ConversionException conversionException = new ConversionException(null);
             wsClientCallBack.onError(conversionException);
         } else if (response != null && response.code() == 402) {
-            InvalidCredentialsException invalidCredentialsException =
-                    new InvalidCredentialsException();
+            InvalidCreedentialsOnPushException invalidCredentialsException =
```

Comment:
It is not necessary to create the new InvalidCreedentialsOnPushException because this error is with HardCoded credentials, InvalidCredentialsException here does not provoke Logout